app/models/manageiq/providers/ibm_cloud/power_virtual_servers/cloud_manager/system_type.rb

```diff
@@ -0,0 +1,13 @@
+class ManageIQ::Providers::IbmCloud::PowerVirtualServers::CloudManager::SystemType < ::Flavor
+  supports :provisioning do
+    if ext_management_system
+      unsupported_reason_add(:provisioning, ext_management_system.unsupported_reason(:provisioning)) unless ext_management_system.supports_provisioning?
+    else
+      unsupported_reason_add(:provisioning, _('not connected to ems'))
+    end
+  end
+
+  def provider_object(_connection = nil)
+    ext_management_system.connect
+  end
```

Comment:
The `provider_object` method is supposed to return the native object, `with_provider_connection` is what would return the connection.

It doesn't look like you're using either though so could probably be dropped